contribs/src/main/java/com/netflix/conductor/contribs/http/GenericHttpTask.java

```diff
@@ -274,15 +278,92 @@ private void setAuthToken(Input input, Workflow workflow) throws Exception {
 		input.getHeaders().put(HttpHeaders.AUTHORIZATION, "Bearer " + response.getAccessToken());
 	}
 
-	void setReasonForIncompletion(HttpResponse response, Task task) {
-		if (response.body != null) {
```

Comment:
We can keep the if-else code as is, collect it into a String and then extract the value if the new param is specified